Doc/library/socket.rst

```diff
@@ -826,8 +826,11 @@ The following functions all create :ref:`socket objects <socket-objects>`.
 
 .. data:: SocketType
 
-   This is a Python type object that represents the socket object type. It is the
-   same as ``type(socket(...))``.
+   This is the base class for a type object of socket objects, and
+   ``isinstance(socket(...), SocketType)`` returns ``True``.
```

Comment:
```suggestion
   This is the base class for socket objects, such that
   ``isinstance(socket(...), SocketType)`` is ``True``.
```

Reply:
Thank you for your professional advice. As we can see, `socket.SocketType` is the base class for the type object `socket.socket`. Consider that elsewhere in this document `socket object` refers to an instance of the type object `socket.socket`. Wouldn't it be less precise if we just said that `SocketType` is the base class of `socket objects`? Looking forward to your more suggestions.